core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-configuration/components/dot-experiments-configuration-scheduling/dot-experiments-configuration-scheduling.component.spec.ts

```diff
@@ -87,6 +87,25 @@ describe('DotExperimentsConfigurationSchedulingComponent', () => {
         expect(store.openSidebar).toHaveBeenCalledOnceWith(ExperimentSteps.SCHEDULING);
     });
 
+    it('should disable button and show tooltip when experiment is nos on draft', () => {
```

Comment:
Do we have any test to check if the tooltip in draft is showing?

Reply:
done